example-app/app/app.module.ts

```diff
@@ -0,0 +1,101 @@
+import { NgModule, } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { BrowserModule } from '@angular/platform-browser';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { RouterModule } from '@angular/router';
+
+import { StoreModule } from '@ngrx/store';
+import { EffectsModule } from '@ngrx/effects';
+import { DBModule } from '@ngrx/db';
+import { StoreRouterConnectingModule } from '@ngrx/router-store';
+import { StoreDevtoolsModule } from '@ngrx/store-devtools';
+import { MaterialModule } from '@angular/material';
+
+import { ComponentsModule } from './components';
+import { BookEffects } from './effects/book';
+import { CollectionEffects } from './effects/collection';
+import { BookExistsGuard } from './guards/book-exists';
+
+import { AppComponent } from './containers/app';
+import { FindBookPageComponent } from './containers/find-book-page';
+import { ViewBookPageComponent } from './containers/view-book-page';
+import { SelectedBookPageComponent } from './containers/selected-book-page';
+import { CollectionPageComponent } from './containers/collection-page';
+import { NotFoundPageComponent } from './containers/not-found-page';
+
+import { GoogleBooksService } from './services/google-books';
+
+import { routes } from './routes';
+import { reducers } from './reducers';
+import { schema } from './db';
+
+
+
+@NgModule({
+  imports: [
+    CommonModule,
+    BrowserModule,
+    BrowserAnimationsModule,
+    MaterialModule,
+    ComponentsModule,
+    RouterModule.forRoot(routes, { useHash: true }),
+
+    /**
+     * StoreModule.provideStore is imported once in the root module, accepting a reducer
+     * function or object map of reducer functions. If passed an object of
+     * reducers, combineReducers will be run creating your application
+     * meta-reducer. This returns all providers for an @ngrx/store
+     * based application.
+     */
+    StoreModule.forRoot(reducers),
+
+    /**
+     * @ngrx/router-store keeps router state up-to-date in the store and uses
+     * the store as the single source of truth for the router's state.
+     */
+    StoreRouterConnectingModule,
+
+    /**
+     * Store devtools instrument the store retaining past versions of state
+     * and recalculating new states. This enables powerful time-travel
+     * debugging.
+     *
+     * To use the debugger, install the Redux Devtools extension for either
```

Comment:
Is the `instrumentOnlyWithExtension()` functionality baked into `instrument()` now?